src/routes/quiz/covers/+page.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+	import { confetti } from '@neoconfetti/svelte';
+	import { enhance } from '$app/forms';
+	import { authStore } from '../../../store/store';
+  
+	export let data;
+	export let form;
+  
+	let user_id = '';
+  
+	authStore.subscribe((store) => {
+	  user_id = store.user.uid;
+	});
+  </script>
+  
+  <style>  
+	.question {
```

Comment:
this styling should be done using tailwind classes

```suggestion
	.class="mt-1 text-center"
```